docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -0,0 +1,1183 @@
+.. highlight:: sh
+
+openSUSE Leap Root on ZFS
+=========================
+
+.. contents:: Table of Contents
+  :local:
+
+Overview
+--------
+
+Caution
+~~~~~~~
+
+- This HOWTO uses a whole physical disk.
+- Do not use these instructions for dual-booting.
+- Backup your data. Any existing data will be lost.
+- This is not an openSUSE official HOWTO page. This document will be updated if Root on ZFS support of 
+  openSUSE is added in the future.
+  Also, `openSUSE's default system installer Yast2 does not support zfs <https://forums.opensuse.org/showthread.php/510071-HOWTO-Install-ZFSonLinux-on-OpenSuse>`__. The method of setting up system 
+  with zypper without Yast2 used in this page is based on openSUSE installation methods written by the 
+  experience of the people in the community.
+  For more information about this, please look at the external links.
+
+
+System Requirements
+~~~~~~~~~~~~~~~~~~~
+
+ 
+- `64-bit openSUSE Leap Live CD w/ GUI (e.g. gnome iso)
+  <https://software.opensuse.org/distributions/leap>`__
+- `A 64-bit kernel is strongly encouraged.
+  <https://github.com/zfsonlinux/zfs/wiki/FAQ#32-bit-vs-64-bit-systems>`__
+- Installing on a drive which presents 4 KiB logical sectors (a “4Kn” drive)
+  only works with UEFI booting. This not unique to ZFS. `GRUB does not and
+  will not work on 4Kn with legacy (BIOS) booting.
+  <http://savannah.gnu.org/bugs/?46700>`__
+
+Computers that have less than 2 GiB of memory run ZFS slowly. 4 GiB of memory
+is recommended for normal performance in basic workloads. If you wish to use
+deduplication, you will need `massive amounts of RAM
+<http://wiki.freebsd.org/ZFSTuningGuide#Deduplication>`__. Enabling
+deduplication is a permanent change that cannot be easily reverted.
+
+Support
+~~~~~~~
+
+If you need help, reach out to the community using the :ref:`mailing_lists` or IRC at
+`#zfsonlinux <irc://irc.freenode.net/#zfsonlinux>`__ on `freenode
+<https://freenode.net/>`__. If you have a bug report or feature request
+related to this HOWTO, please file a new issue and mention `@Zaryob <https://github.com/Zaryob>`__.
+
+Contributing
+~~~~~~~~~~~~
+
+#. Fork and clone: https://github.com/openzfs/openzfs-docs
+
+#. Install the tools::
+
+    sudo apt install python3-pip
+    pip3 install -r docs/requirements.txt
+    # Add ~/.local/bin to your $PATH, e.g. by adding this to ~/.bashrc:
+    PATH=$HOME/.local/bin:$PATH
+
+#. Make your changes.
+
+#. Test::
+
+    cd docs
+    make html
+    sensible-browser _build/html/index.html
+
+#. ``git commit --signoff`` to a branch, ``git push``, and create a pull
+   request. Mention @rlaager.
```

Comment:
Can you remove this mention of me, please?

Reply:
Ok ./